server/utils/rating.util.js

```diff
@@ -9,19 +9,19 @@ class Rating {
    * @returns {number} claps
    */
   static async getArticleClaps(articleId) {
-    const claps = await Clap.findAll({
+    const claps = await Clap.findAndCountAll({
       where: { articleId }
     });
-    return claps;
+    return claps.count;
```

Comment:
This method is `getArticleClaps` and not `getArticleClapsCount`.  Hence, it should return the claps and not the count. @Chiazokam

Reply:
Alright, will change the name of the method

Reply:
@Chiazokam Would it make sense for you to use `findAndCountAll` and then return just the `count`? Wouldn't that result in limiting ourselves and the data?

Reply:
Alright...I understand. Thanks